lib/Makefile

```diff
@@ -54,6 +54,7 @@ install: lib-c lib-a lib-py crit/crit lib/c/criu.pc.in
 	$(Q) install -m 755 lib/c/$(CRIU_A) $(DESTDIR)$(LIBDIR)/$(CRIU_A)
 	$(Q) mkdir -p $(DESTDIR)$(INCLUDEDIR)/criu/
 	$(Q) install -m 644 $(UAPI_HEADERS) $(DESTDIR)$(INCLUDEDIR)/criu/
+	$(Q) cat criu/include/version.h >> $(DESTDIR)$(INCLUDEDIR)/criu/criu.h
```

Comment:
Why don't we explicitly add the

    #include "version.h"

line into criu.h and install the former one into criu/ subdir?

Reply:
Thank you for suggesting that, I updated the PR.